components/session-context.js

```diff
@@ -86,24 +79,18 @@ export const Session = ({ children }) => {
             logout({ returnTo: `${BACKEND_URL}/auth-error` })
             isServerAuthPending.current = false
           } else {
-            // Auth0 and the server authenticated successfully. Get the signed-in session object.
+            // Auth0 and the server authenticated successfully. Set the signed-in session object in
+            // the session context to that any downstream component can retrieve it without doing a
```

Comment:
Should this be "so that any downstream"?

Reply:
Oh yeah — dang. I’ll change that since we’re not urgent.